src/document.rs

```diff
@@ -2,6 +2,7 @@ use std::collections::HashMap;
 use std::collections::hash_map::Entry;
 use std::default::Default;
 use std::path::{Path, PathBuf};
+use std::iter::FromIterator;
```

Comment:
(arbitrary place)
```
Fix EOL issue with tests

Normalizing everything cobalt read to LF and and outputing LF as well. 
excerpts_CRLF test has fixtures as CRLF, and target as LF



typo
```
Could you get this to conform to [conventional changelist format](https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular), including calling out `Fixes #241`?

I didn't point it out in the other PRs because they were purely internal and unrelated to Issues.